cmd/flags.go

```diff
@@ -26,7 +22,7 @@ const (
 	DefaultPlanPath      = "./plan.toml"
 	DefaultProcessType   = "web"
 	DefaultLauncherPath  = "/cnb/lifecycle/launcher"
-	DefaultLogLevel      = logging.InfoLevel
+	DefaultLogLevel      = "info"
```

Comment:
why not reference a constant here?

Reply:
apex/log unfortunately does not expose these constants but we do end up using apex for doing all the validating and parsing. If we were to add our own constants for the each possible value, we would only ever reference `Info`. Knowing those constraints, do you have any strong preferences?